src/SDL.cpp

```diff
@@ -0,0 +1,60 @@
+#include "SDL.h"
+
+#define SCREEN_HEIGHT 768
```

Comment:
Note that this HEIGHT and WIDTH will later have to come from the config file reader